static/docs/install/docker.md

```diff
@@ -0,0 +1,114 @@
+# Using Docker
+
+Currently we don't provide any official Docker image, however you can easily
+create it yourself.
+
+## Writing the Dockerfile
+
+We support several ways to install DVC, the two recommended ones are through
+`pip` and `conda`.
+
+You can base your image on the official Python one:
+
+```dockerfile
+FROM python
+
+# Remember to include the cloud providers that you plan to use,
+# for example: "dvc[s3]"
+RUN pip install dvc
+```
+
+Or use a Conda based image:
+
+```dockerfile
+FROM continuumio/miniconda3
+
+RUN conda install dvc -y --quiet
+```
+
+For a recommended reading about writing Dockerfiles for Python applications, you
+can take a look at https://pythonspeed.com/docker/
+
+## Using Docker for your development environment
+
+Since `dvc` is a command line application, we have several recommendations to
+enhance the experience:
+
+- Make sure your _locale_ is set to UTF-8
+- Install the Zsh shell and its respective autocompletion script.
+- Install `less` so you can pipe the output of `dvc pipeline show`.
+- Enable movements with `Ctrl + Arrow` by editing the _inputrc_ file.
+- Use `pip install dvc` at the end of the file for better caching.
+- Set your `WORKDIR` to the path where you are going to mount your project.
+
+If you are using Linux, remember to map your user to your container. This way,
+you make sure your files are still writable. For more information:
+https://docs.docker.com/engine/security/userns-remap/
+
+### Example Dockerfile
+
+The following is a **suggestion** for a Dockerfile to use during development:
+
+```dockerfile
+FROM python:3.7
+
+# Use UTF-8 as the default locale
+ENV LANG=C.UTF-8
+
+# Better movement with [Ctrl + →] and [Ctrl + ←]
+ENV INPUTRC=/etc/inputrc
+
+RUN set -ex \
+  && echo '"\e[1;5C": forward-word' >> /etc/inputrc \
+  && echo '"\e[1;5D": backward-word' >> /etc/inputrc
```

Comment:
Are these customizations very necessary? They seem like personal preferences.

Reply:
they might be *nice* personal preferences, in which case add a comment saying that they're optional customisation and explaining why.

Reply:
I would just remove if they're not essential.

Reply:
@jorgeorpinel , the only **essential** part is `FROM python` and `RUN pip install dvc` :slightly_smiling_face: 

We are trying to bring a good experience; I'll add a note

Reply:
You're right.